Add `BN Mentorship: Cycle 4 Sign-ups Open` news post (#12652)
ppy/osu-wiki: news/2025/2025-01-10-bn-mentorship-cycle-4.md

---
layout: post
title: "BN Mentorship: Cycle 4 Sign-ups Open"
date: 2025-01-10 12:00:00 +0000
---

A new year, a new cycle of BN Mentorship!

![](/wiki/shared/news/banners/bn-mentorship/banner.jpg)

> For those who aren't on the loop, the BN Mentorship Program is a [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team)-led effort to try and give modders interested in joining the [Beatmap Nominators](/wiki/People/Beatmap_Nominators) the skills they may be lacking to do so. Participants get mentored on how to mod in the way a BN is expected to, given tips on how to optimize their workflow and given crucial insight as to how to avoid common modding pitfalls.

Reviewer: gotta love technicalities (en dash for compounds with spaces)

```suggestion
For those who aren't in the loop, the BN Mentorship Program is a [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team)–led effort to try and give modders interested in joining the [Beatmap Nominators](/wiki/People/Beatmap_Nominators) the skills they may be lacking to do so.

Participants get mentored on how to mod in the way a BN is expected to, with tips on how to optimize their workflow and crucial insight for avoiding common modding pitfalls.
```

Cycles last around 2 months, with this cycle being planned to run from **January 25th** to **March 23rd**.

## Mentors

> Below you'll find a list of mentors for each game mode, alongside their timezones:

Reviewer: ```suggestion
Below you'll find a list of mentors for each game mode, alongside their time zones:
```
